### ISSUE TITLE
About stereotomography

### ISSUE THREAD

**Issue by chenzhaoxingxin** [Lucas-Morais-Freire/AEDII#1]
Hello, it's an honor to be able to search your homepage. I noticed in your introduction that you mentioned that you are currently researching stereotomography. I also want to conduct related research, so I would like to ask if you could share some code for us to learn?

**Comment by Lucas-Morais-Freire** [comment]
Hi, I am glad more people are interested in that field. My research scholarship is currently at its final stages and my professor wants to make the code open for everyone soon (on GitLab though), so I ask for you to wait until our papers are out and the code gets released to the public. It is my professor's code so I don't have permission to show it at the moment, I just parallelized it.

**Comment by chenzhaoxingxin** [comment]
Thank you very much! I want to better understand the relevant formulas in the process of stereotomography through code. Do you do stereotomography based on the adjoint state method or by solving the frechet matrix? How long do you expect to  be able to share the code，can you put a link to gitLab on github, I'm afraid I can't find it or miss it.

Thank you again for sharing the code selflessly. Wish you a smooth work and a happy life!

**Comment by Lucas-Morais-Freire** [comment]
Yeah, so the repo is private as of now, in the coming months we'll make it open as my scholarship approaches its end. But yes we use the adjoint state method.

**Comment by chenzhaoxingxin** [comment]
Dear author:
  Hello, I would like to take the liberty of asking if you have disclosed the code for stereotography! I am very worried that I will miss this valuable learning opportunity!

Thank you again for sharing the code selflessly. Wish you a smooth work and a happy life!
